src/obsidian_ai_hub/coding/orchestrator.py

```
@@ -82,30 +85,60 @@ def _build_messages(
             elif role == "worker":
                 # Worker response provided as system/context observation
                 msgs.append(
-                    SystemMessage(content=f"【CLIワーカーの前回実行結果】\n{content}")
+                    SystemMessage(content=f"【CLIワーカーの実行結果（観測情報）】\n{content}")
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)